release-notes-4.0.0.md

```diff
@@ -1,14 +1,13 @@
-!not-ready-for-release!
 
 #### Version Number
 ${version-number}
 
 #### New Features
 
-#### Known Issues
+* caf-common upgraded to [1.20.0-288](https://github.com/CAFapi/caf-common/releases/tag/v1.20.0)
```

Comment:
What affect does this have to the worker-framework? From a consumer perspective what does this upgrade enable? We should put something here with regards to that rather than simply stating an upgrade has happened.